src/tutorials/ex3.c

```diff
@@ -0,0 +1,186 @@
+
+static char help[] = "Solves a tridiagonal system with lower bound specified as an inequality constraint.\n\
+Sovles finite differences discretizatio of:\n\
```

Comment:
Mistype: Sovles -> Solves

Reply:
@mpecha Let's use suggestions!

```suggestion
Solves finite difference discretization of:\n\
```